waiter/src/waiter/reporter.clj

```diff
@@ -87,15 +100,100 @@
                s/Any s/Any}
               (merge {:pickled? true} config)))
 
+(defn- make-reporting-name
+  "Concatenates elements to form a dotted name for reporting, eliding any null values or empty strings."
+  [prefix & args]
+  (MetricRegistry/name prefix (into-array args)))
+
+(let [df (DecimalFormat. "#.####################")]
+  (defn- format-value
+    "Convert double to string to send to graphite server"
+    [value]
+    (.format df value)))
+
+(defn- report-meter
+  "Report values from a codahale meter"
+  [prefix name ^Metered meter timestamp ^GraphiteSender graphite]
+  (let [send #(.send graphite (make-reporting-name prefix name %1) (format-value (%2 meter)) timestamp)]
+    (send "count" #(.getCount %))
+    (send "m1_rate" #(.getOneMinuteRate %))
+    (send "m5_rate" #(.getFiveMinuteRate %))
+    (send "m15_rate" #(.getFifteenMinuteRate %))
+    (send "mean_rate" #(.getMeanRate %))))
+
+(defn- report-snapshot
+  "Report values from a codahale snapshot"
+  [prefix name ^Snapshot snapshot timestamp ^GraphiteSender graphite]
```

Comment:
On the Waiter routers metrics, we report the one minute rate (`meters/rate-one`). Should we be doing the same here?